.travis.yml

```diff
@@ -6,7 +6,7 @@ cache: bundler
 before_install:
   - gem update --system
   - gem --version
-  - gem install bundler --no-rdoc --no-ri
+  - gem install bundler
```

Comment:
Why removing those? It's better to add (or replace those with) `--no-document` in fact.

Reply:
After giving a thought - I guess it's fine to remove those indeed.